sceptre/scipool/templates/sc-ec2vpc-launchrole.yaml

```diff
@@ -60,6 +61,20 @@ Resources:
                   - "s3:GetObject"
                   - "lambda:InvokeFunction"
                 Resource: '*'
+  ScCreateKmsKeyPolicy:
+    Type: 'AWS::IAM::ManagedPolicy'
+    Properties:
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Sid: AllowCreateKMS
+            Action:
+              - kms:CreateKey
+              - kms:CreateAlias
+              - kms:TagResource
+              - kms:DeleteAlias
```

Comment:
KMS is weird in that it's better to attach the policy to the key directly.
If we add deleteKey here, it would authorize deletion of any KMS key since we don't yet know the arn of the generated key

The delete permission is added here for each individual key:
https://github.com/Sage-Bionetworks/service-catalog-library/pull/244/files#diff-a230fba67f980b82fc1d37fe1efec5b0dd0fb7aca5db47827590853b78fc20d2R317